cmd/qat_plugin/qat_plugin_test.go

```diff
@@ -55,8 +55,7 @@ func TestScanPrivate(t *testing.T) {
 		expectedDevNum  int
 	}{
 		{
-			name:        "Return error for uninitialized device plugin",
-			expectedErr: true,
+			name:       "Return error for uninitialized device plugin",
```

Comment:
The name of the case should better reflect its purpose. Should be something like "No error for uninitialized device plugin".